packages/tutanota-utils/lib/WebAssembly.ts

```diff
@@ -1,5 +1,36 @@
 import { stringToUtf8Uint8Array } from "./Encoding.js"
 
+/*
+ * Functions required to be present inside the Argon2 fallback
+ */
+type Argon2WebAssemblyFallback = {
+	argon2id_hash_raw: Record<string, WebAssembly.ExportValue>
+}
+
+/*
+ * Functions required to be present inside the Liboqs fallback
+ */
+type KyberWebAssemblyFallback = {
+	OQS_KEM_keypair: Record<string, WebAssembly.ExportValue>
```

Comment:
why is it a `Record`? shouldn't it be `OQS_KEM_KEYPAIR_RawFN`?

Reply:
The idea was giving types here as the same type of WebAssembly so there's not need export these types from tuta crypto library. And yes, it should not be a record and just WebAssembly.ExportValue